### DISCUSSION TITLE
How to Perform TheseAction  on iTeamTalk without Using VoiceOver Accesibility Action

### DISCUSSION THREAD

**Discussion by lbk2907** [BearWare/TeamTalk5#2010] [Q&A]
Hey there @bear101, may I ask a few question about iTeamTalk? I need to guide sited people to use TeamTalk. Since I'm a VoiceOver user, some of the button cannot be found by VoiceOver when you swipe left and right but it can be found when using accessibility action. May I know where those button is?

1. How to remove the server from the server list?
2. How to select users to move to another channel?
3. How to join to the channel without need to expand the channel first? I mean, like TeamTalkDroid, there's the join button beside the channel name.

I can done this by using VoiceOver accessibility action but I don't know how to tell the sited PEOPLE that don't use VoiceOver to perform these actions.

**Comment by bear101** [comment]
1. Press the server and navigate to the button of the list of properties and press button called "Delete Server". Note that you cannot delete public and unofficial servers. You can only delete local servers.
2. If you have user-right "Move User" then you'll hear "Move User" as accessible action. Select the action then select a channel and select the accessible action to move user there.
3. Put / in the channel path, e.g. "/channel/subchannel1". Then you join a subchannel.

**Comment by lbk2907** [comment]
I mean how can I done this without using accessible action. I need to guide
sited people. For number 2, are you mean the button is visible on the
screen?

**Reply by bear101** [reply]
https://github.com/BearWare/TeamTalk5/issues/340
